app/config/config.yml

```diff
@@ -66,3 +66,15 @@ swiftmailer:
     username:  "%mailer_user%"
     password:  "%mailer_password%"
     spool:     { type: memory }
+
+# Can be changed directly in the bundle is we found an agreement
+dunglas_action:
+    autodiscover:
+        enabled:   true
+        directories:
+            action: [ ]
+            console: [ Command ]
+    directories:         # List of directories relative to the kernel root directory containing classes to auto-register.
+        action: [ '../src/AppBundle/Controller' ] # Still an autowiring_type definition to add in Symfony to avoid this
```

Comment:
btw, I am cautiously interested about all of this - my criticisms are because if it we *were* to go to the trouble of supporting this, let's do it *really* well.